lib/vrl/compiler/benches/path.rs

```diff
@@ -0,0 +1,45 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion, Throughput};
+use std::fmt;
+use std::str::FromStr;
+use vrl_compiler::path::Path;
+
+struct Parameters {
+    basis: &'static str,
+}
+
+static PARAMETERS: [Parameters; 3] = [
+    Parameters {
+        basis: ".foo.(bar | baz)[1].(qux | quux)",
+    },
+    Parameters {
+        basis: ".foo.(bar | baz)[1].(qux | quux).(a|b)",
+    },
+    Parameters {
+        basis: ".foo.(bar | baz)[1].(qux | quux).(a|b).(c|d)",
+    },
+];
+
+impl fmt::Display for Parameters {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.basis)
+    }
+}
+
+fn benchmark_to_alternative_strings(c: &mut Criterion) {
+    let mut group = c.benchmark_group("path::to_alternative_strings");
```

Comment:
Could we namespace this under `vrl`? That makes the aggregate output of the benchmark suites easier to grok.

Reply:
Sure thing.